src/profiler.h

```diff
@@ -130,7 +130,7 @@ class Profiler {
     Engine* activeEngine();
     Error checkJvmCapabilities();
 
-    time_t addTimeout(time_t start, int timeout);
+    u64 addTimeout(u64 start, int timeout);
```

Comment:
Since `start` and `timeout` are now in different units, add a suffix to make it clear, e.g. `u64 start_micros, int timeout_sec`